ui/types/vault/app-types.ts

```diff
@@ -83,22 +83,6 @@ export interface EngineOwner extends Owner {
   mountPoint: string;
 }
 
-export type SyncDestinationType = 'aws-sm' | 'azure-kv' | 'gcp-sm' | 'gh' | 'vercel-project';
```

Comment:
these were duplicate and exist in the helper folder https://github.com/hashicorp/vault/blob/main/ui/types/vault/helpers/sync-destinations.ts#L0-L1